### ISSUE TITLE
Mini-Class Material: Access Host Machines & Survey (Blue)

### ISSUE THREAD

**Issue by ghost** [Raeltik/red-blue-dev#20]

**Comment by ENatter** [comment]
Access: https://www.makeuseof.com/tag/remote-control-linux-windows/
survey: https://zeltser.com/security-incident-survey-cheat-sheet/

**Comment by SeanLittle4** [comment]
Makes sense. Here is a survey cheat sheet for windows. [windows-cheat-sheet.pdf](https://app.gitkraken.com/api/glo/boards/5cce3394630d92000fecb6ee/attachments/5d0d172df954bf000f8e8a96)
